### DISCUSSION TITLE
zh/reference/documents/introduction_to_streaming_media_related_technologies

### DISCUSSION THREAD

**Discussion by giscus** [ZLMediaKit/docs#10] [Announcements]
# zh/reference/documents/introduction_to_streaming_media_related_technologies

1、流媒体简介 流媒体（streaming media）是指将一连串的媒体数据压缩后，经过网络分段发送数据，在网上即时传输影音以供观赏的一种技术与过程，此技术使得数据包得以像流水一样发送；如果不使用此技术，就必须在使用前下载整个媒体文件，这对于实时性要求比较高的场景而言，显然是不现实的，所以流媒体技术为此孕育而生。 传统的视频监控、IPTV，以及这几年...

https://docs.zlmediakit.com/zh/reference/documents/introduction_to_streaming_media_related_technologies.html

**Comment by nocarethancare** [comment]
After reading this introduction, I have a deep understanding of the development history and pain points of the current streaming media, and I also understand the specific role that zlmediakit can play in the service.

>看完这个介绍 深刻的了解了一下当下流媒体的发展史和痛点 也明白了zlmediakit它具体能在服务中起到什么样的作用

`TRANS_BY_GPT4`
